From 08a544c5176bb00ae88fd5c53267945921f2d046 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 17 May 2023 10:32:11 +0800
Subject: [PATCH 05/12] Revert "drm/panfrost: Fix the
 panfrost_mmu_map_fault_addr() error path"

This reverts commit 8250f7f7ae21a67104e1c8c15ed250b2d01c122d.
---
 drivers/gpu/drm/panfrost/panfrost_mmu.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/panfrost/panfrost_mmu.c b/drivers/gpu/drm/panfrost/panfrost_mmu.c
index c81f3ac22cddb..5ff856ef7d88c 100644
--- a/drivers/gpu/drm/panfrost/panfrost_mmu.c
+++ b/drivers/gpu/drm/panfrost/panfrost_mmu.c
@@ -458,7 +458,6 @@ static int panfrost_mmu_map_fault_addr(struct panfrost_device *pfdev, int as,
 		if (IS_ERR(pages[i])) {
 			mutex_unlock(&bo->base.pages_lock);
 			ret = PTR_ERR(pages[i]);
-			pages[i] = NULL;
 			goto err_pages;
 		}
 	}
-- 
2.25.1

